CMakeLists.txt

```diff
@@ -1,24 +1,28 @@
 
 cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
 
-find_package(IRODS 4.2.11 EXACT REQUIRED)
+#find_package(IRODS 4.2.11 EXACT REQUIRED)
+find_package(IRODS REQUIRED)
 
-set(IRODS_PLUGIN_REVISION "0")
+set(IRODS_PLUGIN_REVISION "1")
 set(IRODS_PLUGIN_VERSION "${IRODS_VERSION}.${IRODS_PLUGIN_REVISION}")
 
+include(IrodsCXXCompiler)
+
 set(CMAKE_C_COMPILER ${IRODS_EXTERNALS_FULLPATH_CLANG}/bin/clang)
 set(CMAKE_CXX_COMPILER ${IRODS_EXTERNALS_FULLPATH_CLANG}/bin/clang++)
 set(CMAKE_EXE_LINKER_FLAGS_INIT "${CMAKE_EXE_LINKER_FLAGS} -stdlib=libc++")
 
-project(irods_netcdf_plugin 
-VERSION "${IRODS_PLUGIN_VERSION}"
-LANGUAGES C CXX)
+#project(irods_netcdf_plugin VERSION "${IRODS_PLUGIN_VERSION}" LANGUAGES C CXX)
+project(irods_netcdf_plugin VERSION "4.3.1.1" LANGUAGES C CXX)
```

Comment:
L16 is correct.

Was it causing issues?

Reply:
It was, but I don't remember what precisely. Will revisit.

Reply:
Seems fine, changing back to what it was.